openpype/plugins/publish/collect_slate_global.py

```diff
@@ -0,0 +1,121 @@
+import os
+import json
+import pyblish.api
+from openpype import resources
+
+
+class CollectSlateGlobal(pyblish.api.InstancePlugin):
```

Comment:
Is it necessary to have collector for the extraction? Content from here can be moved to extractor without any harm, or? Some custom data can be filled afterwards.